code/processes/wdb.q

```diff
@@ -240,21 +240,41 @@ setcompression:{[compression] if[3=count compression;
 				]}
 resetcompression:{setcompression 16 0 0 }
 
+//check if the hdb directory contains current partition
+//if yes check if patition is empty and if it is not see if any of the tables exist in both the 
+//temporary parition and the hdb partition. If there is a clash abort operation otherwise copy 
+//each table to the hdb partition
+movetohdb:{[dw;hw;pt] //break1; 
+  $[not(`$string[pt])in key hsym`$-10 _ hw;
+     .[.os.ren;(dw;hw);{.lg.e[`mvtohdb;"Failed to move data from wdb ",x," to hdb directory ",y," : ",z]}[dw;hw]];
+      not any a[dw]in(a:{key hsym`$x})[hw];
+      [{[y;x] //break2; 
+        $[not(`$last"/"vs x)in key hsym`$y;
+          [.[.os.ren;(x;y);{.lg.e[`mvtohdb;"Table ",(last"/"vs x)," has failed to copy to ",y]}];
```

Comment:
`last"/"vs x` use above, could save as a var on 252 and reuse here possibly (would need to be a projection).